Content.Shared/CriminalRecords/Components/CriminalRecordsConsoleComponent.cs

```diff
@@ -1,14 +1,15 @@
 using Content.Shared.CriminalRecords.Systems;
 using Content.Shared.Radio;
 using Content.Shared.StationRecords;
+using Robust.Shared.GameStates;
 using Robust.Shared.Prototypes;
 
 namespace Content.Shared.CriminalRecords.Components;
 
 /// <summary>
 /// A component for Criminal Record Console storing an active station record key and a currently applied filter
 /// </summary>
-[RegisterComponent]
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
```

Comment:
It seems like you were trying to do prediction for this component but you never properly dirty any of these values. Is there some part of this that i missed?